cmd/gitserver/server/vcs_syncer_git.go

```diff
@@ -36,7 +49,10 @@ func (s *GitRepoSyncer) IsCloneable(ctx context.Context, remoteURL *vcs.URL) err
 			err = ctxerr
 		}
 		if len(out) > 0 {
-			err = errors.Errorf("%s (output follows)\n\n%s", err, out)
+			err = &GitCommandError{
+				Msg:    err.Error(),
+				Output: string(out),
+			}
```

Comment:
Last I checked bytes don't grow on trees, so let's save some:

```suggestion
			err = &GitCommandError{Msg: err.Error(), Output: string(out)}
```

(I'm kidding, ignore me... but also not, you know)